apps/cli/commands/init.py

```
@@ -0,0 +1,150 @@
+    try:
+        # 1. Check system health first
+        console.print("[yellow]Checking system health...[/yellow]")
+        health = asyncio.run(check_system_health())
+
+        if not health["healthy"]:
+            # Report which services failed
+            console.print("[red]❌ System health check failed:[/red]")
+            for service, status in health["services"].items():
+                if not status:
+                    console.print(f"  - {service}: unhealthy")
+            raise typer.Exit(1)
```

Ruff (0.14.1)

Abstract `raise` to an inner function

(TRY301)